src/components/RichTextEditor/Plugins/TextStylePlugin/TextStyleDropdown/DropdownTrigger.tsx

```diff
@@ -1,25 +1,30 @@
+/* eslint-disable react/display-name */
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import React from 'react';
+import React, { ForwardedRef, forwardRef } from 'react';
 import { getPreventDefaultHandler } from '@udecode/plate';
 import { IconCaretDown, IconSize } from '@foundation/Icon';
 import { merge } from '@utilities/merge';
 import { DropdownTriggerProps } from './types';
 import { useSelectedTextStyles } from './useSelectedTextStyles';
 import { AVAILABLE_STYLE_TITLES, DEFAULT_TEXT_STYLE_VALUE } from '../TextStyles';
 
-export const DropdownTrigger = ({ editor, open }: DropdownTriggerProps) => {
+export const DropdownTriggerComponent = (
+    { editor, open, onClick }: DropdownTriggerProps,
+    ref: ForwardedRef<HTMLButtonElement>,
+) => {
     const selectedTextStyles = useSelectedTextStyles(editor);
 
     const label =
```

Comment:
Why dont you put this label also into 'useTextStyleDropdown' and pass label as prop?

Reply:
fixed